WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -60,7 +60,7 @@ + (NSArray *)routes
     [[FBRoute POST:@"/uiaTarget/:uuid/dragfromtoforduration"] respondWithTarget:self action:@selector(handleDrag:)],
     [[FBRoute POST:@"/tap/:uuid"] respondWithTarget:self action:@selector(handleTap:)],
     [[FBRoute POST:@"/keys"] respondWithTarget:self action:@selector(handleKeys:)],
-    [[FBRoute GET:@"/window/:uuid/size"] respondWithTarget:self action:@selector(handleGetWindowSize:)],
```

Comment:
@mykola-mokhnach Can you not remove this endpoint yet? And TODO: to remove it? It will be a lot easier for me to ship it.